iib/workers/tasks/build_merge_index_image.py

```diff
@@ -246,6 +246,20 @@ def handle_merge_request(
                 overwrite_target_index_token,
             )
 
+        _add_label_to_index(
```

Comment:
Do you think we could loop over those to labels?

labels = {'com.redhat.index.delivery.version': prebuild_info['target_ocp_version'], 'com.redhat.index.delivery.distribution_scope': prebuild_info['distribution_scope']}
for label,value in labels.items():
   _add_label_to_index(label,value, temp_dir, 'index.Dockerfile')

Reply:
I'm just trying to keep it consistent with the `Add` request implementation. Please let me know if you feel strongly against it

Reply:
well in create-empty-index I try this approach.